documentation/asciidoc/accessories/camera/camera_hardware.adoc

```diff
@@ -200,6 +220,38 @@ The Raspberry Pi Camera Modules are official products from the Raspberry Pi Foun
 
```

Comment:
I know this is not part of the PR, but this table really ought to be reworked, or preferably removed.  It does not really add any relevant information.

Reply:
If there's general agreement from everyone else I'm totally happy for this to be removed? Any opposition?

Reply:
The information is also quite wrong :)

Reply:
Then it needs to be fixed or go away. Dealer's choice.

Reply:
I vote to get rid of it entirely.

Reply:
Ping @davidplowman. Anyone else that might care?

Reply:
@6by9 and @JamesH65

Reply:
I'm happy for that to go.

Reply:
I did a bit of sleuthing, and the "Hardware Features" table was originally added back in 2014! #93 & #125
So it probably only reflects the V1 camera and not the V2 or HQ cameras?

I agree that it either needs to be fixed or removed, but like Alasdair I'm ambivalent as to which :slightly_smiling_face:

Reply:
So for the avoidance of doubt it's this table that's going, http://127.0.0.1:4000/documentation/accessories/camera.html#hardware-features? 

<img width="876" alt="Screenshot 2021-09-23 at 16 34 00" src="https://user-images.githubusercontent.com/53625/134538126-41d831ce-9bb2-4f49-b940-59420c213b4a.png">

Reply:
Yes, that whole table can be removed!

Reply:
Done, see e2e6b31.